corehq/motech/repeaters/tests/test_models.py

```diff
@@ -49,8 +49,7 @@ def setUp(self):
         conn = ConnectionSettings.objects.create(domain=DOMAIN, name=url, url=url)
         self.repeater = FormRepeater(
             domain=DOMAIN,
-            url=url,
-            connections_settings_id=conn.id
+            url=url
```

Comment:
```suggestion
            onnections_settings_id=conn.id
```

Can we pass `connections_setting_id` here instead of `url`? We have already extracted out these fields from Repeaters to connection settings and they would be totally removed when we move to SQL.

Reply:
We can (which is what I did initially) but there are a number of tests that depend on it not being set.

Prior to this PR it was setting `connections_settings_id` which has the same effect as not setting it due to the misspelling:
```diff
- connections_setting_id
+ connection_setting_id
```

Reply:
Ohh! I think we have fixed those tests in SQLRepeaters PR but they are not merged yet. 
Thanks for clarifying, Simon 😄